chainntnfs/neutrinonotify/neutrino.go

```diff
@@ -528,6 +528,8 @@ func (n *NeutrinoNotifier) historicalConfDetails(confRequest chainntnfs.ConfRequ
 		regFilter, err := n.p2pNode.GetCFilter(
 			*blockHash, wire.GCSFilterRegular,
 			neutrino.NumRetries(5),
+			neutrino.OptimisticReverseBatch(),
+			neutrino.MaxBatchSize(scanHeight-startHeight+1),
```

Comment:
Maybe note that this won't end up fetching the whole batch every iteration due to a cache being used.

Reply:
Done.